drivers/cleanup.py

```diff
@@ -1532,13 +1532,21 @@ def __init__(self, uuid, xapi, createLock, force):
             if force:
```

Comment:
Question. Do we want to attempt the retry if "force" is given, to avoid spuriously giving the "not attatched, ignoring" message?

If so, perhaps it would be cleaner to test for self.xapi.isPluggedHere() before the sleep() in wait_for_plug(), and just call wait_for_plug() here.

Reply:
I think we'd need to review the use cases for force and assess if they're even valid anymore. My feeling is that they're unused at best and dangerous at worst and we might want to remove this to prevent people shooting themselves in their feet.